docs/pact-reference.md

```diff
@@ -974,7 +972,7 @@ Strings also support multiline by putting a backslash before and after whitespac
 Symbols are string literals representing some unique item in the runtime, like a function or a table name.
 Their representation internally is simply a string literal so their usage is idiomatic.
 
-Symbols are created with a preceding tick, thus they do no support whitespace or multiline.
+Symbols are created with a preceding tick, thus they do not support whitespaces or multilines.
```

Comment:
whitespace and multiline should not be pluralized, and are idiomatic when discussing strings. If you must, go literate with "whitespace characters" and "multi-line syntax".